index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<header class="header">
+    <div class="top-menu">
+        <i class="fas fa-bars" onclick="openNav()"></i>
+        <a href='#' class="logo">NEWSROOM</a>
+        <div>
+            <i class="fas fa-search" onclick="toggleSearch()"></i>
+        </div>
+    </div>
+    <div class='search-box'>
+        <form class="form">
+            <input type='text' class="search" name="search" placeholder="Search..." style="width: 0px">
+        </form>
+    </div>
+
+    <div id="sidenav" class="sidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
```

Comment:
Avoid href="javascript:void(0)". If you need a clickable element, use button. Also inline event listener :(

Reply:
thx - replaced function with event listener, and changed element to <i>